runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -48,6 +48,54 @@ extern TR::Monitor *assumptionTableMutex;
 
 uint32_t serverMsgTypeCount[JITaaS::J9ServerMessageType_ARRAYSIZE] = {};
 
+static UDATA
+findField(J9VMThread *vmStruct, J9ConstantPool *constantPool, UDATA index, BOOLEAN isStatic, J9Class **declaringClass)
```

Comment:
In the future we may want to export this from the VM and call it directly.
Please add a comment where you took this method from.